### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ZachAttacksProblems**
Update to indicate notebooks should be cleared off outputs before submitting a PR.

### CODE REVIEW

**Comment by byrcyb** [comment]
Dang, the second I wrote my previous comment on PR#219 I knew I should have double checked our contributing guide.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Contributing Guidelines
 
-Create a branch off of master for your task. After completing your notebook(s) and content review, make the recommended changes and open a pull request from your branch to master.
+Create a branch off of master for your task. After completing your notebook(s) and content review, make the recommended changes and open a pull request from your branch to master. 
+
+We consider it a best practice to clear all cell outputs before commiting. This can be done with 'Kernel > Restar & Clear Output' (or 'Edit > Clear All Outputs' in JupyterLab) from the top menu bar.
```
Comment (byrcyb) [inline CONTRIBUTING.md:5]: one small typo "Restar" -> "Restart"

**Review by byrcyb — CHANGES_REQUESTED** [review]
Thanks for adding this in. I could have sworn it was in this doc at some point, but, regardless, I'm happy you did.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ZachAttacksProblems) [reply CONTRIBUTING.md]: Fixed typo.

Comment (byrcyb) [reply CONTRIBUTING.md]: Awesome, thanks!

**Review by byrcyb — APPROVED** [review]
Thanks for fixing the typo and updating the doc.

**Pull request merged**